awx/api/serializers.py

```diff
@@ -1780,11 +1753,8 @@ class Meta:
                   'groups_with_active_failures', 'has_inventory_sources')
 
     @property
-    def show_capabilities(self):  # TODO: consolidate in 3.3
-        if self.version == 1:
-            return ['copy', 'edit', 'start', 'schedule', 'delete']
-        else:
-            return ['copy', 'edit', 'delete']
+    def show_capabilities(self):
+        return ['copy', 'edit', 'delete']
```

Comment:
This, as well as the other similar method, no longer needs to be a `@property` it can just be

```python
show_capabilities = ['copy', 'edit', 'delete']
```